backend/packages/Upgrade/.env.test

```diff
@@ -19,7 +19,7 @@ LOG_OUTPUT=dev
 # PostgreSQL DATABASE
 #
 TYPEORM_CONNECTION=postgres
-TYPEORM_HOST=postgres
+TYPEORM_HOST=localhost
```

Comment:
Can you revert this back to 'postgres'. This is just required for local testing.